parse_lib.py

```diff
@@ -106,6 +106,12 @@ def update_anchor_href(anchor):
         anchor['target'] = '_blank'
 
 
+def convert_to_img(html, equation):
```

Comment:
On an unrelated note, I think we could improve the naming for this function.  In particular, I think the naming semantics are crossing "levels of abstractions" here.  Although, in the once place where `convert_to_img` is invoked, the second argument is an `equation`, this function really should work on any SVG that is represented as an `<object>`.  Thus, this lower level function should not know or care that the SVGs it operates on happen to all be equations.  This abstraction leakage is bad for a number of reasons:

1. It makes it more difficult to understand what `convert_to_img` is doing---initially I assumed that it was converting equations into images, and that it depended on the argument being an image.
2. It makes the function less likely to be re-used when appropriate; e.g. what if we need to use it on SVGs that are NOT equations?  In that case, the user would either not know that they could use this function, or they would have to update the variable names, or they would be lazy and use it anyway, despite the (now deceptive) variable names.